src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -385,5 +385,9 @@ static String getResource(String key) {
 				{"R_kerberosLoginFailed", "Kerberos Login failed: {0} due to {1} ({2})"},
 				{"R_StoredProcedureNotFound", "Could not find stored procedure ''{0}''."},
 				{"R_jaasConfigurationNamePropertyDescription", "Login configuration file for Kerberos authentication."},
+				{"R_SQLVariantSupport", "sql_variant datatype is not supported in pre-SQL 2008 version"},
+                {"R_invalidProbbytes", "sql_variant: invalid probBytes for {0} type."},
```

Comment:
Needed your input for this. Is it better to throw exception of check with assert?